### ISSUE TITLE
Core components dead on fresh installation

### ISSUE THREAD

**Issue by jschaeke** [domogik/domogik#182]
Got message below on Admin panel on first startup, core components are dead. Can't find the cause of it. I am running on an arm Debian Wheezy (cubox first model, 1GB ram, running on SSD esata drive)

'Ooups, it seems that you have an issue with your current configuration!
The message queue is not working for PUB/SUB messages.
This is related to the component MQ forwarder.
The related configuration file is : /etc/domogik/domogik-mq.cfg.'
netstat -lapute | grep 404 : https://gist.github.com/jschaeke/4fcf8887e1019d7c421e
https://drive.google.com/file/d/0B2xiU5oodx7EcDBtWVBxOUczRWM/view
https://drive.google.com/file/d/0B2xiU5oodx7EdHpfV2t2NmVyYm8/view?usp=sharing

**Comment by fritz-smh** [comment]
Can you download this file in domogik sources : https://raw.githubusercontent.com/domogik/domogik/develop/diagnostic.py

And can you execute it and give us the output ?

Thanks

**Comment by jschaeke** [comment]
sure, but it will be probably for tomorrow or day after..

**Comment by fritz-smh** [comment]
ok, no issue :)

**Comment by fritz-smh** [comment]
by the way what is your system ? processor, memory, etc ?
Can you also look at load average when domogik is started ?

**Comment by jschaeke** [comment]
It's also a fresh install of Debian Weezy without any additional software but Domogik on a Cubox first gen (https://en.wikipedia.org/wiki/CuBox). This little box, that was in that time a faster cpu than the raspi 1, has 1GB of ram and it is running a hardfp debian build. The storage is a 64GB SSD.   I will use top to see the load on startup.

**Comment by fritz-smh** [comment]
Thanks, could you also look at the amound of free ram ?

Domogik is consuming a lot of RAM currently, but 1Go should be enough as
some people used successfully domogik on a rpi 2

**Comment by jschaeke** [comment]
Made me thinking, I don't think I created a swap partition... Will check that first.

**Comment by jschaeke** [comment]
Ok I created a swap and had to reinstall whole debian again because the shrinking of the main partition didn't end well.... this time I had a succesful installation! I noticed during installation the memory usage floats around 900MB with peaks over 1GB (the swap was used). After installation, just starting the domogik admin takes 440mb of ram. Haven't installed domoweb yet. Thanks for all your assistance.
